packages/cx-elements/index.tailwind.css

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright © 2019 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;500;700&display=swap');
```

Comment:
@wodenx  This import isn't going through and going to site level..  I was playing in package.json to get this file included but its not working.

Reply:
I wouldn't expect this file to be used.  We currently use a site-level [/styles/style.css](https://github.com/hvanyo/Bodiless-JS/blob/feature/bc-typo/sites/__cxstarter__/src/css/style.css) which is included in the site-level [gatsby-browser.js](https://github.com/hvanyo/Bodiless-JS/blob/feature/bc-typo/sites/__cxstarter__/gatsby-browser.js), though I propose we move this out of site level (maybe into gatsby-theme-bodiless, or a new gatsby-theme-canvasx).

But need to discuss the best way tot add teh font, since it will need to be removed at site level for any site which uses a different font.
- zarbees does it at site level in [gatsby-ssr](, as  post-body element (which is better for performance)
- we could try a dynamic import in a token (so that sites could override that part of the token)?

Maybe we build a canvasx-theme which takes some fonts as a gatsby plugin option, and adds them.  This could also hndle including the index.tailwind.css... 
-